src/NCDialog.cc

```diff
@@ -20,6 +20,9 @@
 
    Author:     Michael Andres <ma@suse.de>
 
+# just to make the y2makepot script happy:
```

Comment:
`#` looks a bit strange. why not have it without it?

Reply:
The idea was to make a bit different than the `textdomain` line to see that it is really a comment. But  I was also not sure it it was not confusing. I'll remove it.